site/hi.njk

```diff
@@ -0,0 +1,123 @@
+                <li class="pb-2">फोन कॉल या वीडियो चैट जैसे दूरदराज के विकल्पों के साथ संभव के रूप में कई सामाजिक इंटरैक्शन को बदलें।</li>
+                <li class="pb-2">अपने घर से बाहर न निकलें जब तक कि बिल्कुल आवश्यक न हो।</li>
+            </ol>
+
+            <p class="text-xl">
+                कृपया ध्यान रखें कि कोई सही या गलत राशि नहीं है। केवल उन कार्यों को करें जिन्हें आप सहज महसूस करते हैं और जो आपकी आजीविका को खतरे में नहीं डालते हैं। <strong>इस पर अपनी नौकरी मत छोड़ो!</strong> हालांकि, ध्यान रखें कि हर क्रिया मदद करती है।
+            </p>
+
+            <h2 class="text-3xl">यह इतना महत्वपूर्ण क्यों है?</h2>
+
+            <p>
+                SARS-CoV-2 एक अत्यधिक संक्रामक और संभावित घातक वायरस है जो एक श्वसन रोग का कारण बनता है जिसे COVID-19 कहा जाता है। आप इसे इसके कई अन्य नामों में से एक के तहत जान सकते हैं, जिसमें 2019-nCoV, उपन्यास कोरोनावायरस, वुहान कोरोनावायरस, चीन या वुहान फ्लू या सिर्फ कोरोनवायरस शामिल हैं। ये सभी एक ही वायरस को संदर्भित करते हैं जिसे यह आंदोलन रोकने की कोशिश कर रहा है।
+            </p>
+
+            <p>
+                पिछले महीनों और हफ्तों में, वायरस और इसी बीमारी की तुलना इन्फ्लूएंजा वायरस और मौसमी फ्लू से अधिक की गई है। लक्षणों में ओवरलैप और समान रूप से गंभीरता के कारण, यह तुलना स्वाभाविक रूप से बहुत सारे लोगों के लिए आती है, हालांकि, इस नए वायरस के बारे में अब तक जो भी हम जानते हैं, उसे देखते हुए, यह खतरा है कि समाज के लिए खतरा आसानी से अलग नहीं होता है।
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12 font-bold">
+                <li class="pb-2">
+                    <h3>फ्लू से अधिक संक्रामक</h3>
+                    <p class="font-normal">
+                        एक अनुमानित R0 के साथ (जनसंख्या में सीधे एक मामले में उत्पन्न होने वाले मामलों की अपेक्षित संख्या जहां सभी व्यक्ति संक्रमण के लिए अतिसंवेदनशील होते हैं) 1.4 &ndash; 6.49 और 3.28<sup><a class="source-link" href="https://academic.oup.com/jtm/advance-article/doi/10.1093/jtm/taaa021/5735319" target="_blank">[1]</a></sup> का एक अनुमानित अनुमान , SARS-CoV-2 बहुत अधिक संक्रामक और है मौसमी फ्लू की तुलना में बहुत तेजी से फैलता है, जिसमें 1.28<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/25186370" target="_blank">[2]</a></sup> का माध्य R0 होता है ।
+                    <p>
+                </li>
+                <li class="pb-2">
+                    <h3>फ्लू से अधिक घातक</h3>
+                    <p class="font-normal">
+                        SARS-CoV-2 की तथाकथित केस घातक दर (CFR) लगभग 2%<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/32146445" target="_blank">[3]</a></sup> अनुमानित है, जिसका अर्थ है कि COVID-19 से पीड़ित होने वाले अनुमानित 2% लोग इसके शिकार होंगे। इसकी तुलना में मौसमी इन्फ्लूएंजा का सीएफआर लगभग 0.1%<sup><a class="source-link" href="https://www.cdc.gov/flu/about/burden/2018-2019.html" target="_blank">[4]</a></sup> होने का अनुमान है , जिसका अर्थ है कि SARS-CoV-2 का अनुमान मौसमी फ्लू से लगभग 20 गुना अधिक घातक है।
```

Comment:
`                        SARS-CoV-2 की तथाकथित केस घातक दर (सीएफआर) लगभग 2%<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/32146445" target="_blank">[3]</a></sup> अनुमानित है, जिसका अर्थ है कि COVID-19 से पीड़ित होने वाले अनुमानित 2% लोग इसके शिकार होंगे। इसकी तुलना में मौसमी इन्फ्लूएंजा का सीएफआर लगभग 0.1%<sup><a class="source-link" href="https://www.cdc.gov/flu/about/burden/2018-2019.html" target="_blank">[4]</a></sup> होने का अनुमान है , जिसका अर्थ है कि SARS-CoV-2 का अनुमान मौसमी फ्लू से लगभग 20 गुना अधिक घातक है।
`

Edited.

Reply:
Changed. Thank you.